src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/XmlFormMetadataLoader.php

```diff
@@ -62,10 +60,10 @@ public function __construct(
 
     public function getMetadata(string $key, string $locale, array $metadataOptions = []): ?FormMetadata
     {
-        $configCache = $this->getConfigCache($key);
+        $path = $this->getCachePath($key);
 
-        if ($configCache->isFresh()) {
-            $formMetadata = @include $configCache->getPath();
+        if (\file_exists($path)) {
```

Comment:
no `file_exists` just `@include ()` we should not access the filesystem in case of debug = false.